### ISSUE TITLE
1.4.2: test suite is failing in two units

### ISSUE THREAD

**Issue by kloczek** [xiph/flac#540]
cmake settings (`cmake -L`)
```console
-- Cache values
BUILD_CXXLIBS:BOOL=ON
BUILD_DOCS:BOOL=ON
BUILD_DOXYGEN:BOOL=ON
BUILD_EXAMPLES:BOOL=ON
BUILD_PROGRAMS:BOOL=ON
BUILD_SHARED_LIBS:BOOL=ON
BUILD_TESTING:BOOL=ON
BUILD_UTILS:BOOL=OFF
CMAKE_BUILD_TYPE:STRING=RelWithDebInfo
CMAKE_INSTALL_PREFIX:PATH=/usr
ENABLE_64_BIT_WORDS:BOOL=OFF
ENABLE_WERROR:BOOL=OFF
HAVE_GIT:FILEPATH=/usr/bin/git
HAVE_PANDOC:FILEPATH=/usr/bin/pandoc
INSTALL_CMAKE_CONFIG_MODULE:BOOL=ON
INSTALL_MANPAGES:BOOL=ON
INSTALL_PKGCONFIG_MODULES:BOOL=ON
WITH_ASM:BOOL=ON
WITH_AVX:BOOL=ON
WITH_FORTIFY_SOURCE:BOOL=ON
WITH_OGG:BOOL=ON
WITH_STACK_PROTECTOR:BOOL=ON
```
and test suite is failing
```console
+ cd flac-1.4.2
+ /usr/bin/ctest --test-dir x86_64-redhat-linux-gnu --output-on-failure --force-new-ctest-process -j48
Internal ctest changing into directory: /home/tkloczko/rpmbuild/BUILD/flac-1.4.2/x86_64-redhat-linux-gnu
Test project /home/tkloczko/rpmbuild/BUILD/flac-1.4.2/x86_64-redhat-linux-gnu
    Start 1: libFLAC
    Start 2: libFLAC++
    Start 3: grabbag
    Start 4: flac
    Start 5: metaflac
    Start 6: replaygain
    Start 7: seeking
    Start 8: streams
    Start 9: compression
1/9 Test #9: compression ......................***Failed    0.02 sec
Using FLAC binary : /home/tkloczko/rpmbuild/BUILD/flac-1.4.2/x86_64-redhat-linux-gnu/src/flac/flac
/home/tkloczko/rpmbuild/BUILD/flac-1.4.2/test/test_compression.sh: line 30: noisy-sine.wav: No such file or directory

2/9 Test #6: replaygain .......................***Failed    0.13 sec
Generating stream...
Testing FLAC replaygain 8000 (8000 x 1) ...  \c
ERROR: can't open input file rpg-tone-8000.wav: No such file or directory

3/9 Test #3: grabbag ..........................   Passed    0.59 sec
4/9 Test #2: libFLAC++ ........................   Passed    0.91 sec
5/9 Test #5: metaflac .........................   Passed    1.31 sec
6/9 Test #1: libFLAC ..........................   Passed    2.54 sec
7/9 Test #7: seeking ..........................   Passed    2.70 sec
8/9 Test #4: flac .............................   Passed   20.74 sec
9/9 Test #8: streams ..........................   Passed  545.05 sec

78% tests passed, 2 tests failed out of 9

Total Test time (real) = 545.09 sec

The following tests FAILED:
          6 - replaygain (Failed)
          9 - compression (Failed)
Errors while running CTest
```
On fist look it may be some some side effect of parallelisation because
```console
[tkloczko@devel-g2v flac-1.4.2]$ find . -name noisy-sine.wav
./x86_64-redhat-linux-gnu/test/noisy-sine.wav
```

**Comment by kloczek** [comment]
Strange .. after exclude those two units anither one popped out
```console
+ cd flac-1.4.2
+ /usr/bin/ctest --test-dir x86_64-redhat-linux-gnu --output-on-failure --force-new-ctest-process -j48 ' ' -E '^(replaygain|version)'
Internal ctest changing into directory: /home/tkloczko/rpmbuild/BUILD/flac-1.4.2/x86_64-redhat-linux-gnu
Test project /home/tkloczko/rpmbuild/BUILD/flac-1.4.2/x86_64-redhat-linux-gnu
    Start 1: libFLAC
    Start 2: libFLAC++
    Start 3: grabbag
    Start 4: flac
    Start 5: metaflac
    Start 6: seeking
    Start 7: streams
    Start 8: compression
1/8 Test #8: compression ......................***Failed    0.02 sec
Using FLAC binary : /home/tkloczko/rpmbuild/BUILD/flac-1.4.2/x86_64-redhat-linux-gnu/src/flac/flac
/home/tkloczko/rpmbuild/BUILD/flac-1.4.2/test/test_compression.sh: line 30: noisy-sine.wav: No such file or directory

2/8 Test #3: grabbag ..........................   Passed    0.50 sec
3/8 Test #2: libFLAC++ ........................   Passed    0.90 sec
4/8 Test #5: metaflac .........................   Passed    1.18 sec
5/8 Test #1: libFLAC ..........................   Passed    2.40 sec
6/8 Test #6: seeking ..........................   Passed    2.53 sec
7/8 Test #4: flac .............................   Passed   17.62 sec
8/8 Test #7: streams ..........................   Passed  498.98 sec

88% tests passed, 1 tests failed out of 8

Total Test time (real) = 499.00 sec

The following tests FAILED:
          8 - compression (Failed)
Errors while running CTest
```

**Comment by ktmf01** [comment]
The problem is running the tests in parallel, that was never meant to be possible. Can you try the fix in #541?

**Comment by kloczek** [comment]
Thx. 
Any pans to release new version soon? 🤔

**Comment by ktmf01** [comment]
Not really, no. I would like to make some strides in the foreign metadata handling and do a release when these features are thoroughly tested. As far as I know there aren't really any fixes since 1.4.2 that people are waiting for, so I'm not in a hurry, really.
